dracut/30ignition/module-setup.sh

```diff
@@ -61,6 +61,13 @@ install() {
             "$systemdsystemunitdir/ignition-$x.target"
     done
 
+    # For consistency tear down the network between the initramfs and
+    # real root. See https://github.com/coreos/fedora-coreos-tracker/issues/394#issuecomment-599721763
+    inst_script "$moddir/coreos-teardown-initramfs-network.sh" \
+	"/usr/sbin/coreos-teardown-initramfs-network"
```

Comment:
Minor: looks like tab here instead of spaces.

Reply:
fixed